plugins/defaultplugins/ifplugin/model/stn/keys_agent_stn.go

```diff
@@ -0,0 +1,12 @@
+package stn
+
+// KeyPrefix returns the prefix used in ETCD to store vpp STN config
+func KeyPrefix() string {
+	return "vpp/config/v1/stn/rules/"
```

Comment:
optionally you can define string constant "vpp/config/v1/stn/rules/" and return it here
then in func Key you can return that constant + ruleName